### ISSUE TITLE
Document in README that the Xournal++ file needs to be saved prior to recognition

### ISSUE THREAD

**Issue by PellelNitram** [PellelNitram/xournalpp_htr#4]
The present issue is related to the problem that arised in this issue: #3 .

---

In principle, there is a safeguard already in place to inform the user that the file needs to be saved first, see [here](https://github.com/PellelNitram/xournalpp_htr/blob/master/plugin/main.lua#L32).

However, there are two problems:

1. The safeguard fails because `document_structure['xoppFilename']` fails when `xoppFilename` is not present in `document_structure`. Hence, the safeguard is completely useless by construction.
2. Document in the README that the user should first save the file.

Cheers!

**Comment by PellelNitram** [comment]
tl;dr: I am closing this issue because I resolved it.

Context:
- It turns out that point 1 is not a problem.
- I documented in the README that the Xournal++ file first needs to be saved, see commit e35d86f.
